core/templates/pages/collection-editor-page/services/collection-editor-state.service.spec.ts

```diff
@@ -207,7 +207,7 @@ describe('Collection editor state service', function() {
       fakeEditableCollectionBackendApiService,
       'fetchCollectionAsync').and.callThrough();
 
-    CollectionEditorStateService.loadCollection(5);
+    CollectionEditorStateService.loadCollectionAsync(5);
```

Comment:
Are these expected changes?

Reply:
Yes @kevintab95 these changes are expected in these files, and I've checked that this file is not assigned to anyone.
Thanks.

Reply:
This file is not listed in the original issue thread (https://github.com/oppia/oppia/issues/10306). Let's restrict changes to read-only-collection-backend-api.service as mentioned in your PR title and description. Thanks.

Reply:
Done, PTAL.
Thanks